examples/se_2_3_localization.cpp

```diff
@@ -24,16 +27,16 @@
  *  We consider a robot in 3D space surrounded by a small
  *  number of punctual landmarks or _beacons_.
  *  The robot is assumed to be mounted with an IMU whose
- *  measurements are fed as exogeneous inputs to the system,
- *  and is able to measure the location
+ *  measurements are fed as exogeneous inputs to the system.
+ *  The robot is able to measure the location
  *  of the beacons w.r.t its own reference frame.
  *  We assume in this example that the IMU frame coincides with the robot frame.
  *
  *  The robot extended pose X is in SE_2(3) and the beacon positions b_k in R^3,
  *
- *      X = |  R   t  v|              // position, orientation and linear velocity
- *          |  0   1  0|
- *          |  0   0  1|
+ *      X = |    R   p  v|              // position, orientation and linear velocity
+ *          |0 0 0   1  0|
+ *          |0 0 0   0  1|
```

Comment:
I do not think that these leading zeros are necessary, but ok

Reply:
Removed in https://github.com/artivis/manif/pull/154/commits/ded121b75f0f401c518336cf0d31ad862be50275